### PR TITLE
Docs: Add Swedish Guide for Pokemon Emerald

### PR DESCRIPTION
**Pull request by Tsukino-uwu**
## What is this fixing or adding?
Swedish setup guide, setup_sv.md

## How was this tested?
https://markdownlivepreview.com/

### CODE REVIEW

**Comment by ThePhar** [comment]
This might also be good to find someone in the AP discord who's also fluent in Swedish to do a review.

**Comment by Tsukino-uwu** [comment]
> This might also be good to find someone in the AP discord who's also fluent in Swedish to do a review.

Asked in a old thread for a review, not sure what else i can really do for finding someone

File: worlds/pokemon_emerald/docs/setup_sv.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+1. Pokemon Emerald använder Archipelago's BizHawk Klient. Om klienten fortfarande inte startat efter att du patchat ditt spel,
+så kan du bara öppna den igen från launchern.
+2. Dubbel kolla att EmuHawk faktiskt startat med den patchade ROM filen.
+3. I EmuHawk, gå till `Tools > Lua Console`. Denna Lua konsolen får inte stängas medans du spelar.
+4. I Lua konsolen, Tryck på `Script > Open Script…`.
+5. Leta reda på din Archipelago map och i den öppna `data/lua/connector_bizhawk_generic.lua`.
+6. Emulatorn och klienten kommer så småningom koppla ihop med varanda. I klienten kommer du sedan kunna see om allt är
+kopplat och att Pokemon Emerald startat ordentligt.
+7. För att ansluta klienten till en server, skriv in din lobby adress och port i text fältet tex,`archipelago.gg:38281`
```
Comment (Zunawe) [inline worlds/pokemon_emerald/docs/setup_sv.md:57]:
```suggestion
7. För att ansluta klienten till en server, skriv in din lobby adress och port i text fältet tex, `archipelago.gg:38281`
```

File: worlds/pokemon_emerald/docs/setup_sv.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+- Ett Engelsk Pokémon Emerald ROM, Archipelago kan inte hjälpa dig med detta
+- [BizHawk](https://tasvideos.org/BizHawk/ReleaseHistory) 2.7 eller senare
+
+### konfigurera BizHawk
+
+När du har installerat Bizhawk, öppna `EmuHawk.exe` och ändra följande inställningar:
+
+- Om du använder Bizhawk 2.7 eller 2.8, gå till `Config > Customize`. sen på "advanced tab" bytt Lua core från
```
Comment (Zunawe) [inline worlds/pokemon_emerald/docs/setup_sv.md:13]:
```suggestion
- Om du använder BizHawk 2.7 eller 2.8, gå till `Config > Customize`. sen på "advanced tab" bytt Lua core från
```

**Review by Zunawe — COMMENTED** [review]
Thanks for the translation.

Many lines are >120 characters, so the line breaks should be put in better spots to keep under that stylistic limit.

There are a few lines with leading or trailing spaces. Not that it would really have an impact. But it'd be better if they weren't there.

I see a bunch of words that I'd expect to be capitalized that aren't or vice versa. And I see places I'd expect periods/full stops where there aren't any. But I'm only going off my experience with other languages I know, not Swedish.

Running the doc through a translator back to English, there's nothing obviously problematic. But again, I don't speak Swedish at all.

**Comment by Tsukino-uwu** [comment]
Added proposed changes made by @bittersweetrin , they did a pull request to my branch as they didn't have permission to do one here and/or make a review
"touchup - proposed changes for the Swedish translation of Emerald setup"

**Review by bittersweetrin — APPROVED** [review]

File: worlds/pokemon_emerald/docs/setup_sv.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+filen är installerad i din BizHawk-mapp.
+
+Om du bara tänkt spela själv och du inte bryr dig om automatisk spårning eller ledtrådar, så kan du stanna här, stänga av
+klienten, och starta ditt patchade ROM med valfri emulator. Dock, för multvärldsfunktionen eller 
+andra Archipelago-funktioner, fortsätt nedanför med BizHawk.
+
+## Anslut till en server
+
+Om du vanligtsvis öppnar en patchad fil så görs steg 1-5 automatiskt åt dig. Även om det är så, kom ihåg dessa steg ifall
+du till exempel behöver stänga ner och starta om något medans du spelar.
```
Comment (Zunawe) [inline worlds/pokemon_emerald/docs/setup_sv.md:49]:
```suggestion
Om du vanligtsvis öppnar en patchad fil så görs steg 1-5 automatiskt åt dig. Även om det är så, kom ihåg dessa steg
ifall du till exempel behöver stänga ner och starta om något medans du spelar.
```

File: worlds/pokemon_emerald/docs/setup_sv.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+3. Öppna `ArchipelagoLauncher.exe`
+4. Välj "Open Patch" på vänstra sidan, och välj din patchfil.
+5. Om detta är första gången du patchar, så kommer du behöva välja var ditt ursprungliga ROM är.
+6. En patchad `.gba` fil kommer skapas på samma plats som patchfilen.
+7. Första gången du öppnar en patch med BizHawk-klienten, kommer du också behöva bekräfta var `EmuHawk.exe`
+filen är installerad i din BizHawk-mapp.
+
+Om du bara tänkt spela själv och du inte bryr dig om automatisk spårning eller ledtrådar, så kan du stanna här, stänga av
+klienten, och starta ditt patchade ROM med valfri emulator. Dock, för multvärldsfunktionen eller 
+andra Archipelago-funktioner, fortsätt nedanför med BizHawk.
```
Comment (Zunawe) [inline worlds/pokemon_emerald/docs/setup_sv.md:44]:
```suggestion
7. Första gången du öppnar en patch med BizHawk-klienten, kommer du också behöva bekräfta var `EmuHawk.exe` filen är
installerad i din BizHawk-mapp.

Om du bara tänkt spela själv och du inte bryr dig om automatisk spårning eller ledtrådar, så kan du stanna här, stänga
av klienten, och starta ditt patchade ROM med valfri emulator. Dock, för multvärldsfunktionen eller andra
Archipelago-funktioner, fortsätt nedanför med BizHawk.
```

File: worlds/pokemon_emerald/docs/setup_sv.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Pokémon Emerald har en fullt fungerande spårare med stöd för automatisk spårning.
+
+1. Ladda ner [Pokémon Emerald AP Tracker](https://github.com/seto10987/Archipelago-Emerald-AP-Tracker/releases/latest)
+och
+[PopTracker](https://github.com/black-sliver/PopTracker/releases).
+2. Placera tracker pack zip-filen i packs/ där du har PopTracker installerat.
+3. Öppna PopTracker, och välj Pokemon Emerald.
+4. För att automatiskt spåra, tryck på "AP" symbolen längst upp.
+5. Skriv in Archipelago-serverns uppgifter (Samma som du använde för att ansluta med klienten), "Slot"-namn samt lösenord.
```
Comment (Zunawe) [inline worlds/pokemon_emerald/docs/setup_sv.md:77]:
```suggestion
5. Skriv in Archipelago-serverns uppgifter (Samma som du använde för att ansluta med klienten), "Slot"-namn samt
lösenord.
```

**Review by Zunawe — APPROVED** [review]
From a technical perspective, looks fine. Ran the webhost and looked at the page and clicked a few links.

Suggestions cover some line length violations.

The above approval was given as a result of this conversation, for context: https://discord.com/channels/731205301247803413/1161741293039394936/1251143162739691561

[A PR](https://github.com/Tsukino-uwu/Archipelago/pull/1) with suggestions/corrections was made to the branch in lieu of a "proper" review here, and that PR was merged.

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**